src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -216,10 +219,10 @@ public String updateDeviceMember(ManagerDeviceMemberElement deviceElement, Strin
         deviceMember.setId(Long.parseLong(deviceElement.getId()));
         deviceMember.setName(name);
         updateMember(deviceMember, vserverIpAddress, contactIpAddress, ipAddress, gateway, prefixLength);
-        return updateDeviceMember(deviceMember, Long.parseLong(this.vs.getMgrId()));
+        return updateDeviceMember(deviceMember, Long.parseLong(this.vs.getMgrId()), true);
```

Comment:
Shouldnt this be in the create instead? For create you could always use the default values if none is provided. For updates you can always override even if null is provided. This should address the osc creation path.